### ISSUE TITLE
"val" as a keyword is not highlighted in the editor

### ISSUE THREAD

**Issue by joey-coleman** [symphonytool/symphony#94]
> The following bug was [originally reported](https://sourceforge.net/p/compassresearch/bugs/94/) on Sourceforge by [pglvdm](https://sourceforge.net/u/pglvdm/profile/), 2013-10-07 13:48:57.233000:

fixed in [e9e93d7192a0c4d7e1266b92970e65a603f8fba3]

**Comment by joey-coleman** [comment]
> [Comment](https://sourceforge.net/p/compassresearch/bugs/94/#2ce9) by [joeycoleman](https://sourceforge.net/u/joeycoleman/profile/), 2013-10-09 07:51:36.667000:
- **assigned_to**: Kenneth Lausdahl

**Comment by joey-coleman** [comment]
> [Comment](https://sourceforge.net/p/compassresearch/bugs/94/#d38b) by [lausdahl](https://sourceforge.net/u/lausdahl/profile/), 2013-10-15 07:37:45.885000:
- Description has changed:

Diff:

```

--- old
+++ new
@@ -0,0 +1 @@
+fixed in [e9e93d7192a0c4d7e1266b92970e65a603f8fba3]

```
- **status**: open --> closed
